volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -43,104 +69,222 @@ def _generator(self):
 
         # Read in the Symbol File
         symbol_table = intermed.IntermediateSymbolTable.create(
-            context=self.context,
-            config_path=self.config_path,
+            context=context,
+            config_path=config_path,
             sub_path="windows",
             filename="mft",
-            class_types={"FILE_NAME_ENTRY": mft.MFTFileName, "MFT_ENTRY": mft.MFTEntry},
+            class_types={
+                "FILE_NAME_ENTRY": mft.MFTFileName,
+                "MFT_ENTRY": mft.MFTEntry,
+                "ATTRIBUTE": mft.MFTAttribute,
+            },
         )
 
         # get each of the individual Field Sets
         mft_object = symbol_table + constants.BANG + "MFT_ENTRY"
         attribute_object = symbol_table + constants.BANG + "ATTRIBUTE"
-        si_object = symbol_table + constants.BANG + "STANDARD_INFORMATION_ENTRY"
-        fn_object = symbol_table + constants.BANG + "FILE_NAME_ENTRY"
 
         # Scan the layer for Raw MFT records and parse the fields
-        for offset, _rule_name, _name, _value in layer.scan(
-            context=self.context, scanner=yarascan.YaraScanner(rules=rules)
+        for offset, _, _, _ in layer.scan(
```

Comment:
Not sure why you're removing the descriptive names here.  If anyone comes looking at the code it'll tell them what's returned for the sake of a few characters?  It's ok, but I don't understand why it was done when they were already properly annotated?

Reply:
I actually thought it was a coding bug to have the name like that if it isn't used. Pylint definitely yelled at me about it in the past.

Reply:
Possibly, but I don't think black complains and I prefer it as a way of ensuring people know what the parameters are in case they want to start using them, rather than trying to rely on memory, getting it wrong and spending cycles just because we wanted to drop a few characters in a commit...  5;)

Reply:
changed it back